docs: add upgrade note at the top of changelog (#5349)
confluentinc/ksql: CHANGELOG.md

# Change Log

## Upgrading

> If you're upgrading from a ksqlDB version < 0.7, please make sure to read and follow these [upgrade instructions](./docs-md/operate-and-deploy/installation/upgrading.md).

Reviewer: ```suggestion
If you're upgrading from a ksqlDB version before 0.7, follow these [upgrade instructions](./docs-md/operate-and-deploy/installation/upgrading.md).
```
